cms/djangoapps/contentstore/views/videos.py

```diff
@@ -171,7 +171,8 @@ def get_profile_header(profile):
 
     profile_whitelist = VideoUploadConfig.get_profile_whitelist()
 
-    videos = list(_get_videos(course))
+    videos_data = _get_videos(course, request)
+    videos = videos_data["results"]
```

Comment:
I would leave the download functionality as it for now. i.e. don't get the paginated response.